tests/phpt/exceptions/with_finally/02_finally_with_throw.php

```diff
@@ -0,0 +1,18 @@
+@kphp_should_fail
+/construct not implemented/
+<?php
+
+function inverse($x) {
+    if (!$x) {
+        throw new Exception('Division by zero.');
+    }
+    return 1/$x;
+}
+
+try {
+    echo inverse(0);
```

Comment:
There's no difference between 02_finally_with_throw.php and 03_finally_without_throw.php
These compiler tests are negative, so compiler doesn't know anything about runtime, that is why no matter which number is argument of `inverse` call